jsapp/scss/stylesheets/partials/_registration.scss

```diff
@@ -1,5 +1,8 @@
 @use "scss/_variables";
 
+// A rare light mid-blue
+$kobo-light-mid-blue: #c0e3ff;
```

Comment:
Is this Ignacio approved? :D Because maybe it should go to `colors` (in `kobo-common` too)?

Reply:
Actually I have another request. I see looking at `kobo-common` project that we've decided to update the `kobo-mid-blue` color value to `$kobo-mid-blue: hsl(206deg 100% 88%); // #c0e3ff`. So could you please:
1. remove `$kobo-light-mid-blue` that you've defined here
2. update `$kobo-mid-blue` in `/jsapp/scss/_colors.scss` to have 88%
3. use `$kobo-mid-blue` here instead :heart: